client/landing/stepper/declarative-flow/internals/global.scss

```diff
@@ -149,8 +149,69 @@ button {
 	}
 }
 
+/**
+ * Tailored flow stylings
+ */
 .newsletter,
 .link-in-bio {
+	button.components-button {
+		border-radius: 4px;
+	}
+
+	button.site-icon-with-picker__upload-button {
+		font-family: 'SF Pro Text', $sans;
+		transition: ease 300ms;
+		border: 1px solid var( --studio-gray-10 );
+
```

Comment:
we lost the border-radius here. From what I see this should be a circular button 
<img width="139" alt="Screenshot 2022-08-25 at 12 49 53" src="https://user-images.githubusercontent.com/7000684/186645822-4e49782c-d4b8-42ea-9a13-fa229e409562.png">

Reply:
Also the hover state background doesn't look good when we have an image
<img width="289" alt="Screenshot 2022-08-25 at 12 51 23" src="https://user-images.githubusercontent.com/7000684/186646077-9d67b6e0-62ed-4631-b924-6bb8dd92d7f0.png">